cpp/src/parquet/column_writer.h

```diff
@@ -143,7 +143,8 @@ class PARQUET_EXPORT ColumnWriter {
 
   static std::shared_ptr<ColumnWriter> Make(ColumnChunkMetaDataBuilder*,
                                             std::unique_ptr<PageWriter>,
-                                            const WriterProperties* properties);
+                                            const WriterProperties* properties,
+                                            BloomFilter* bloom_filter = NULLPTR);
```

Comment:
Should we avoid default parameters?

Reply:
`ColumnWriter` is an `PARQUET_EXPORT`, so it's export to user and we need to considering the compatiblility. should I add a new function here?